tests/cypress/integration/Store/storePage.cy.js

```diff
@@ -0,0 +1,72 @@
+import { GetPluginId, getAppId } from '../wp-module-support/pluginID.cy';
+import { comingSoon } from '../wp-module-support/utils.cy';
+
+const pluginId = GetPluginId();
+const appId = getAppId();
+
+describe( 'Store Page- WooCommerce is deactivated/uninstalled', () => {
```

Comment:
@sangeetha-nayak What happens in `Buy Now` scenario for Woocommerce?

Reply:
I can add it as a new test in next set of tests.